snapcraft/cli/_errors.py

```diff
@@ -23,6 +23,7 @@
 from . import echo
 from snapcraft.config import CLIConfig as _CLIConfig
 from snapcraft.internal import errors
+from snapcraft.internal.lxd import _errors as lxd_errors
```

Comment:
If packages outside of lxd need access to `_errors`, then it should be `errors` (no underscore). You can also optionally import `ContainerSnapcraftCmdError` into lxd's `__init__.py` if you prefer not to make the entire module public.